src/screens/EventDetailsScreen/IconButton.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from "react";
+import { Image, StyleSheet } from "react-native";
+import Touchable from "../../components/Touchable";
+
+type Props = {
+  onPress: Function,
+  source: Image.propTypes.source
+};
+
+const IconButton = ({ onPress, source }: Props) => (
+  <Touchable onPress={onPress}>
```

Comment:
This needs an `accessibilityLabel`. Otherwise the screen reader will not read any text for this button.